srcpkgs/nsnake/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'nsnake'
+pkgname=nsnake
+version=3.0.1
+revision=1
+wrksrc=nSnake-${version}
```

Comment:
`wrksrc="nSnake-${version}"`

Reply:
not necessary at all once you change the pkgname.

Reply:
@Vaelatern why are you saying that to me? I'm not the one who chose the pkgname.

Reply:
because the change you proposed will be unnecessary when the package name is changed to follow upstream

Reply:
@Vaelatern @maxice8 oh, it's okay, sorry, I just misunderstood the discussion! ;)